kodev

```diff
@@ -811,16 +809,18 @@ TARGET:
                     kodev-release --ignore-translation android
                     assert_ret_zero $?
                 fi
-                setup_env
+                if [ -n "${KODEBUG}" ]; then
+                    KODEBUG_SUFFIX=-debug
+                fi
                 # clear logcat to get rid of useless cruft
                 adb logcat -c
                 # uninstall existing package to make sure *everything* is gone from memory
                 # no assert_ret_zero; uninstall is allowed to fail if there's nothing to uninstall
-                adb uninstall "org.koreader.launcher"
+                adb uninstall "org.koreader.launcher${KODEBUG_SUFFIX/-/.}"
```

Comment:
Since when do we have a .debug? I thought we only had the F-Droid flavor thing.

Reply:
You mean something about gradle saying `Debug` there makes it put .debug at the end?

Reply:
Yes, it builds a debug build (signed with a insecure key), with a debug suffix in the package name (it does not conflict with the regular package)

Reply:
That was never an issue for me, hence why I didn't include it when I wrote the entire debug build infrastructure. :-)

Anyway, good to know.